test/entity.ts

```diff
@@ -19,6 +19,7 @@ import * as sinon from 'sinon';
 import {Datastore} from '../src';
 import {Entity} from '../src/entity';
 import {IntegerTypeCastOptions} from '../src/query';
+import {decode} from 'punycode';
```

Comment:
Is this used anywhere, I think it might just be `vscode` adding something?

Reply:
Not used. Will delete.